src/connect.js

```diff
@@ -20,7 +20,17 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  let obj = {};
+  if (params !== undefined) {
+    obj = params;
+  }
+  return (extraParams) => {
```

Comment:
you can write simply
```suggestion
  return (extraParams) => extraParams ? { ...obj, ...extraParams } : obj;
```